Parsers/ASimAuditEvent/Parsers/ASimAuditEventMicrosoftWindowsEvents.yaml

```diff
@@ -24,70 +24,180 @@ ParserParams:
     Default: false
 ParserQuery: |
   let parser = (disabled:bool = false) {
-    union 
-    (
-        union 
-        (
+    // Parsed Events Ids
+    let ParsedEventIds = dynamic([1102,4698,4699,4700,4701,4702,4929,5025,5027,5028,5029,5030,5034,5035,5037,7035,7036,7040,7045,2009,5136]);
+    // Eventlog Event Ids
+    let EventlogEventIds = dynamic([1102]);
+    // Scheduled Task Event Ids
+    let ScheduledTaskEventIds = dynamic([4698,4699,4700,4701,4702]);
+    // Active Directory Replica Source Naming Context Event Ids
+    let ActiveDirectoryReplicaIds = dynamic([4929]);
+    // Firewall Event Ids
+    let FirewallEventIds = dynamic([5025,5027,5028,5029,5030,5034,5035,5037]);
+    // Service Event Ids
+    let ServiceEventIds = dynamic([7035,7036,7040,7045,2009]); 
+    // Directory Service Object Ids
+    let DirectoryServiceIds = dynamic([5136]);   
+    // EventID Lookup
+    let EventIDLookup = datatable(EventID:int, Operation:string, EventType:string, Object:string, ObjectType:string, EventResult:string)
+    [   
+        1102, "Delete Logs", "Delete", "Security Logs", "Event Log", "Success",
+        4698, "Create Scheduled Task", "Create", "", "Scheduled Task", "Success",
+        4699, "Delete Scheduled Task", "Delete", "", "Scheduled Task", "Success",
+        4700, "Enable Scheduled Task", "Enable", "", "Scheduled Task", "Success",
+        4701, "Disable Scheduled Task ", "Disable", "", "Scheduled Task", "Success",
+        4702, "Update Scheduled Task", "Set", "", "Scheduled Task", "Success",
+        4929, "Remove Active Directory Replica Source Naming Context", "Delete", "", "Other", "Success",
+        5025, "Stop Firewall Service", "Disable", "Firewall Service", "Service", "Success",
+        5027, "Retrieve the Security Policy From The Local Storage", "Read", "Firewall Service", "Service", "Failure",
+        5028, "Parse the new Security Policy", "Set", "Firewall Service", "Service", "Failure",
+        5029, "Initialize the Firewall Driver", "Initialize", "Firewall Service", "Service", "Failure",
+        5030, "Start the Firewall Service", "Start", "Firewall Service", "Service", "Failure",
+        5034, "Stop Firewall Driver", "Stop", "Firewall Driver", "Driver", "Failure",
+        5035, "Start Firewall Driver", "Start", "Firewall Driver", "Driver", "Failure",
+        5037, "Terminating Firewall Driver", "Terminate", "Firewall Driver", "Driver", "Failure",
+        7035, "Start Control Sent", "Execute", Service "Service", "Success",
```

Comment:
Value should be in double quotes. This is causing the KQL Validation failure.

Reply:
done